libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1487,9 +1483,11 @@ GCS_MAVLINK::update_receive(uint32_t max_time_us)
         bool parsed_packet = false;
 
         // Try to get a new message
-        if (mavlink_parse_char(chan, c, &msg, &status)) {
-            hal.util->persistent_data.last_mavlink_msgid = msg.msgid;
-            packetReceived(status, msg);
+        if (mavlink_parse_char(chan, c, NULL, NULL)) {
```

Comment:
nullptr, otherwise static analyzer spam with warning

Reply:
This is a call into the C library bindings.  static analyzer should see the markers in the relevant header and be good with it.